### DISCUSSION TITLE
Initialization Strategies

### DISCUSSION THREAD

**Discussion by jvanderwee** [getsentry/sentry-kotlin-multiplatform#112] [Ideas]
Have you considered this approach for working around the need for the context on Android?

https://proandroiddev.com/how-to-avoid-asking-for-android-context-in-kotlin-multiplatform-libraries-api-d280a4adebd2

**Comment by buenaflor** [comment]
Thank you for the suggestion, haven't seen that yet! It definitely looks very promising. I will investigate if this is a viable approach for us

**Reply by buenaflor** [reply]
Update: @jvanderwee This suggestion is currently in review #117 and implements the approach as suggested in the blog. The only difference is that we currently decide against using the `startup` lib and can accomplish the same with a simple `ContentProvider`.
